### ISSUE TITLE
cannot import name 'PY3' from 'torch._six' on fresh install of bolts

### ISSUE THREAD

**Issue by davzaman** [PyTorchLightning/pytorch-lightning-bolts#581]
## 🐛 Bug

Fresh install of pytorch-lightning and pytorch-lightning bolts leads to error `cannot import name 'PY3' from 'torch._six'`

Looks like it was renamed to `PY37` in `torch._six`. I changed all references to `PY3` to `PY37` inside `pl_bolts/datasets/imagenet_dataset.py` and the problem resolved.

```
Traceback (most recent call last):
  File "debugtuner.py", line 4, in <module>
    from pl_bolts.datamodules import MNISTDataModule
  File "/home/davina/miniconda3/envs/ap/lib/python3.8/site-packages/pl_bolts/__init__.py", line 47, in <module>
    from pl_bolts import callbacks, datamodules, datasets, losses, metrics, models, optimizers, transforms, utils
  File "/home/davina/miniconda3/envs/ap/lib/python3.8/site-packages/pl_bolts/datamodules/__init__.py", line 2, in <module>
    from pl_bolts.datamodules.binary_mnist_datamodule import BinaryMNISTDataModule
  File "/home/davina/miniconda3/envs/ap/lib/python3.8/site-packages/pl_bolts/datamodules/binary_mnist_datamodule.py", line 4, in <module>
    from pl_bolts.datasets import BinaryMNIST
  File "/home/davina/miniconda3/envs/ap/lib/python3.8/site-packages/pl_bolts/datasets/__init__.py", line 11, in <module>
    from pl_bolts.datasets.imagenet_dataset import extract_archive, parse_devkit_archive, UnlabeledImagenet
  File "/home/davina/miniconda3/envs/ap/lib/python3.8/site-packages/pl_bolts/datasets/imagenet_dataset.py", line 12, in <module>
    from torch._six import PY3
ImportError: cannot import name 'PY3' from 'torch._six' (/home/davina/miniconda3/envs/ap/lib/python3.8/site-packages/torch/_six.py)
```

#### Code sample
`from pl_bolts.datamodules import MNISTDataModule`

### Expected behavior

the package should run with no error.

### Environment

 - PyTorch Version (e.g., 1.0): 1.8
 - OS (e.g., Linux): MacOs
 - How you installed PyTorch (`conda`, `pip`, source): conda
 - Build command you used (if compiling from source): pip install
 - Python version: 3.8
 - CUDA/cuDNN version: 10.1
 - GPU models and configuration: 4 nvidia gpus
 - Any other relevant information:

**Comment by Borda** [comment]
@davzaman mind share what you mean fresh install, from git/master, or from pip?

**Comment by davzaman** [comment]
Sure, I installed from conda actually. Here's all the steps used to initialize my new conda environment:
```
conda -create -n mynewenv
conda activate mynewenv
conda install pytorch cudatoolkit=10.1 -c pytorch
pip install xgboost tensorboardX ray[tune] rich guildai miceforest
conda install pytorch-lightning -c conda-forge
conda install pyarrow orange3 scikit-learn numpy pandas tqdm
```

**Comment by Borda** [comment]
@davzaman I do not see how you install bolts...

**Comment by davzaman** [comment]
Oh I'm so sorry i completely missed that line!`pip install pytorch-lightning-bolts`

**Comment by Borda** [comment]
ok, can you try `pip install https://github.com/PyTorchLightning/pytorch-lightning-bolts/archive/master.zip`

**Comment by davzaman** [comment]
I created a new env and instead of `pip install pytorch-lightning-bolts` I ran what you suggested and tried importing the MNISTDataModule and it did not produce errors.

**Comment by Borda** [comment]
so it was fixed with #579
